### PR TITLE
Test generic server

### PR DESCRIPTION
**Pull request by dbox-fnal**

### CODE REVIEW

File: src/tests/test_generic_server.py
Comment (renbauer-fnal) [inline src/tests/test_generic_server.py:69]: I think the most important call here is self.csc.new_config_obj.new_config_msg(), can we add a check for that? Seems like it should be easy with MockCSC?

File: src/tests/test_generic_server.py
Comment (renbauer-fnal) [inline src/tests/test_generic_server.py:118]: Is the third arg here intentional?

File: src/tests/test_generic_server.py
Comment (renbauer-fnal) [inline src/tests/test_generic_server.py]: I would prefer to use mock.ANY here instead of the specific message text. I think it makes sense to ensure an alarm is fired but having to keep the unit test's message text exactly right is a bit, sort of like over-fitting. WDYT?

**Review by renbauer-fnal — CHANGES_REQUESTED** [review]

File: src/tests/test_generic_server.py
Comment (dbox-fnal) [reply src/tests/test_generic_server.py:118]: It was intentional.  Changed it to self.assertEqual(nm, 'FOO', "expected %s=='FOO'" % nm) for better readability

File: src/tests/test_generic_server.py
Comment (dbox-fnal) [reply src/tests/test_generic_server.py:69]: added check

File: src/tests/test_generic_server.py
Comment (dbox-fnal) [reply src/tests/test_generic_server.py]: your suggestion is much more robust.  Done

**Comment by dbox-fnal** [comment]
review suggestions all implemented.  Waiting for approval

**Review by renbauer-fnal — APPROVED** [review]

**Pull request merged**
